src/field.rs

```diff
@@ -197,11 +197,15 @@ impl FieldElement {
     }
 
     /// Given a nonzero field element, compute its inverse.
+    ///
     /// The inverse is computed as self^(p-2), since
     /// x^(p-2)x = x^(p-1) = 1 (mod p).
-    ///
-    /// XXX should we add a debug_assert that self is nonzero?
+    //
+    // XXX do we want the debug assertion to check for zero? it breaks behaviour
+    // such as that such as in curve25519_dalek::montgomery::test::identity_to_monty.
```

Comment:
not relevant for this PR, but I guess we should probably specify in all of our code (inv, invsqrt, sqrt_ratio) how we handle zeros

Reply:
yeah we should make an issue to note that